cloud-agent/service/server/src/main/scala/org/hyperledger/identus/agent/server/jobs/PresentBackgroundJobs.scala

```diff
@@ -839,21 +839,19 @@ object PresentBackgroundJobs extends BackgroundJobsHelper {
                       case Base64(data) =>
                         val base64Decoded = new String(java.util.Base64.getDecoder.decode(data))
                         val verifiedClaims = for {
-                          sdJwtPresentationPayload <- ZIO.fromEither(base64Decoded.fromJson[SdJwtPresentationPayload])
-                          iss <- ZIO.fromEither(sdJwtPresentationPayload.presentation.iss)
+                          presentation <- ZIO.succeed(PresentationJson(base64Decoded))
+                          iss <- ZIO.fromEither(presentation.iss)
                           ed25519PublicKey <- resolveToEd25519PublicKey(iss)
                           verifiedClaims = SDJWT.getVerifiedClaims(
                             IssuerPublicKey(ed25519PublicKey),
-                            sdJwtPresentationPayload.presentation,
-                            sdJwtPresentationPayload.claimsToDisclose.toJson
+                            presentation
                           )
                           _ <- ZIO.logInfo(s"ClaimsValidationResult: $verifiedClaims")
-                          _ <- ZIO.logInfo(s"ClaimsValidationResult: ${sdJwtPresentationPayload.claimsToDisclose}")
                           result: SDJWT.ClaimsValidationResult =
                             verifiedClaims match {
                               case validClaims: SDJWT.ValidClaims =>
                                 validClaims.verifyDiscoseClaims(
-                                  sdJwtPresentationPayload.claimsToDisclose.asObject.getOrElse(Json.Obj())
+                                  Json.Obj()
```

Comment:
What is the purpose of the presentation with no claims in sd-jwt?

Reply:
The claims to disclosed are in the presentation, we only need presentation to disclose the claims